src/ucp/api/ucp_def.h

```diff
@@ -554,6 +561,56 @@ typedef ucs_status_t (*ucp_am_callback_t)(void *arg, void *data, size_t length,
                                           ucp_ep_h reply_ep, unsigned flags);
 
 
+/**
+ * @ingroup UCP_ENDPOINT
+ * @brief Callback to process incoming Active Message sent by
+ * @ref ucp_am_send_nbx routine.
+ *
+ * @param [in]  arg           User-defined argument.
+ * @param [in]  header        Active message header.
+ * @param [in]  header_length Active message header length in bytes.
```

Comment:
if header==null do we have to set header_length to zero ?

Reply:
@brminich this comment has not been addressed

Reply:
@shamisp, fixed